website/blog/2024-01-22-charting-libraries-d3-nvd3-upgrade.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Charting Libraries(D3 & NVD3) Upgraded - Action Required"
+author: Bhargavi Gunda
+---
+---
+<!-- truncate -->
+
+| Current version | Updating to |
+|-----------------|-------------|
+| D3 3.5.7	       | 7.8.5       |
+| NVD3 1.0.0      | 1.8.6       |
+
+# Introduction
+Exciting news for WaveMaker Web Apps users.The upcoming WaveMaker Web Apps Charting libraries upgrade, scheduled for Release 11.5.3 in the fourth week of January 2024, 
+brings significant enhancements to both D3.js and NVD3. 
```

Comment:
enhancements from D3.js upgrade? 

When you say "to"  it will mean we made changes to d3.js